### DISCUSSION TITLE
Paid consultation for expertise/validation of a custom routing solution for https://tripleaxle.com

### DISCUSSION THREAD

**Discussion by cdbattags** [valhalla/valhalla#4184] [General]
Hi folks,

First, feel free to shut this down immediately if this is against the "Discussions" rules/manners of this project.

I'm co-founder and CTO of a heavy haul trucking SaaS product called **Triple Axle** (https://tripleaxle.com). We're currently in the process of standing up a custom routing service using OSM and Valhalla which will include added rules for some of the more unknown or had to find truck restrictions for over-dimensional (height, width, length, weight) truck routing profiles.

Some of these rules include:
- every interstate in a specific state having maximum width limit
- bridge weight limits
- multi lane calculations for width
- various temporary restrictions from the state for height, width, length, weight usually based on construction

Our timeline is extremely aggressive with a hopeful release in September. We currently have Valhalla with US and Canada data up and running and I'm in the process of standing up a custom version of the OSM Rails application (https://github.com/openstreetmap/openstreetmap-website) that will allow us to track specific edits and export just those as separate PBF dumps on top of the Geofabrik Canada and US data.

I'm looking for a paid consultation from anyone with experience in this type of work. Also, **_I'm not looking for code! Just a chat!_**

If you or anyone you know might be a good fit for this type of discussion, please reach out to me at:

**christian@tripleaxle.com**

We fully intend to contribute all our data back into OSM as soon as we find a pathway to name our OSM node tags appropriately for these type of restrictions.

**Comment by kevinkreiser** [comment]
no harm in asking the question thanks for handling it with care! i'll let anyone with interest contact you directly through your email here. best of luck and let us know if you find any issues with valhalla proper along the way :smile:
